declarative-examples/buildPluginParentPOM.groovy

```diff
@@ -0,0 +1,29 @@
+pipeline {
+    tools {
+        maven "apache-maven-3.1.0"
+        jdk "default"
+    }
+
+    agent {
+        label "master"
+    }
+
+    stages {
+        stage("build") {
+            steps {
+                dir("tmp") {
+                    git changelog: false, poll: false, url: 'git://github.com/jenkinsci/plugin-pom.git', branch: 'master'
```

Comment:
This is a weird example, fwiw - the test this is coming from is explicitly cloning a second repo to build. It's probably not a good example to use here as is - i.e., remove the tmp dir stuff and the checkout entirely, since that's very specific to the test case.